### ISSUE TITLE
Doesn't export, Shows 33% progress, never completes 17.10

### ISSUE THREAD

**Issue by fatriff** [ozmartian/vidcutter#141]
Use case, 1 minute mp4 (lib264) Full HD clip, cut 3 segments out then save video.. Progress indicator on each cut shows 33% but it never goes further (cursor spinning when over application), sys mon shows ffmpeg as running for a few seconds then vanishes from process tree. No output in the selected output folder.

Same with & without smart cut enabled.

Version 5.5.0

**Comment by jeekajoo** [comment]
same here

**Comment by Baemir** [comment]
Yeah I have the same problem on both linux and windows. Actually this app has never worked for me :/

**Comment by landerson78** [comment]
I have this issue but it stops at 60 percent

**Comment by dustinkerstein** [comment]
Same here on Windows x64 and ViCutter 5.50 on every video I've tried recently.

**Comment by Dirwinc** [comment]
It's giving me the same problem on every cut no matter what setting or cut length selected.

**Comment by fatriff** [comment]
I upgraded from Kubuntu 17.10 to 18.04 yesterday and thought i'd give vidcutter another try but still nothing, it still will not render the output and just remains on 0% now.

**Comment by rickst29** [comment]
This is probably duplicate of another issue: vidcutter spawns a separate ffmpeg task for each exported clip. They all fight against each other to acquire sytem memory from the "free" portion. In a system with less than 16GB memory, OR a video with lots of clips, 'export' will hang, waiting on the child 'ffmpeg' processes. Each of the ffmpeg jobs keep trying to get more memory and fails do get it, creating infinite loops with high CPU.

**Comment by harsha-main** [comment]
I'm able to resolve this issue on my Ubuntu. Check my answer here.
https://github.com/ozmartian/vidcutter/issues/171#issuecomment-864420860

**Comment by ozmartian** [comment]
Please try latest Launchpad PPA w/ Ubuntu 20+ or Snap/Flatpak varieties. These issues should be resolved.
